migrations/20200207195019_projectList.js

```diff
@@ -0,0 +1,43 @@
+exports.up = async function(knex) {
+    await knex.schema.createTable('projects', (project)=>{
+        project.increments()
+        project.string('name').notNullable()
+        project.string('description')
+        project.boolean('completed').notNullable().defaultTo(0)
+    })
+
+    await knex.schema.createTable('resources', (resource)=>{
+        resource.increments()
+        resource.string('name').notNullable().unique()
+        resource.string('description')
+        resource.integer('project_id').notNullable().references('id').inTable('projects')
```

Comment:
Question here

Reply:
i suggest not to reference a foreign key because it is not necessary and would more then likely not satisfy a user experience on the front end